### ISSUE TITLE
RuntimeError: In function void opencl::verifySupported(af::dim4) [with int rank = 3] when I run py.test

### ISSUE THREAD

**Issue by gitansh95** [QuazarTech/Poisson_Solver#1]
When I run py.test, it gives me the following error :

```
============================================================= test session starts =============================================================
platform linux -- Python 3.6.1, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /home/gitansh95/work/git/Poisson_Solver, inifile:
collected 0 items / 1 errors 

=================================================================== ERRORS ====================================================================
___________________________________________________ ERROR collecting test_fft_poisson_3d.py ___________________________________________________
test_fft_poisson_3d.py:66: in <module>
    test_fft_poisson()
test_fft_poisson_3d.py:55: in test_fft_poisson
    Ex_numerical, Ey_numerical, Ez_numerical = fft_poisson(rho, dx, dy, dz)
fft_poisson_3d.py:33: in fft_poisson
    rho_hat = af.fft3(rho)
../../../anaconda3/lib/python3.6/site-packages/arrayfire/signal.py:223: in fft3
    c_dim_t(dim0), c_dim_t(dim1), c_dim_t(dim2)))
../../../anaconda3/lib/python3.6/site-packages/arrayfire/util.py:79: in safe_call
    raise RuntimeError(to_str(err_str))
E   RuntimeError: In function void opencl::verifySupported(af::dim4) [with int rank = 3]
E   In file src/backend/opencl/fft.cpp:68
E   Invalid argument at index 1
E   Expected: isSupLen(dims[i])
--------------------------------------------------------------- Captured stdout ---------------------------------------------------------------
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=========================================================== 1 error in 1.88 seconds ===========================================================
```

**Comment by gitansh95** [comment]
Added `af.set_backend('cpu')` as the first line in function `test_fft_poisson()` in `test_fft_poisson_3d.py` as a temp fix.

**Comment by shyams2** [comment]
Fixed in #4
